index.php

```diff
@@ -25,20 +25,36 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
 	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="<?php echo $lang ?>" lang="<?php echo $lang ?>">
+<!-- TODO FIND PERMANENT PROPER SOLUTION FOR THIS. DO NOT MERGE / DEPLOY. -->
+<div id="WMDE-Banner-Container"
+     data-trackingparams="piwik_campaign=wpde-02-181120&amp;piwik_kwd=wpde-02-181120-top-ctrl"
+     data-campaign-tracking="wpde-02-181120"
+     data-tracking="wpde-02-181120-top-ctrl"
```

Comment:
Right now the banners rely on these fields being set through this container element, we have to adapt the banners so that they do not rely on this anymore. If this element is not present, the JavaScript code dies during execution.